packages/core/src/lib/TwaManifest.ts

```diff
@@ -100,6 +101,8 @@ export class TwaManifest {
   generatorApp: string;
   webManifestUrl?: URL;
 
+  private static log: Log = new Log('twa-manifest');
```

Comment:
nit: I've been passing a log object in the constructor, but initialising with a default value. This helps with tests so those don't pollute the console. But I'm ok to keep like this too.

Reply:
It's being used from within a static function though. I could make it a module-level constant but that won't help with the pollution issue.